test/ssc_test/cmod_battery_pvsamv1_test.cpp

```diff
@@ -93,8 +93,8 @@ TEST_F(CMPvsamv1BatteryIntegration_cmod_pvsamv1, ResidentialACBatteryModelIntegr
 
 	ssc_number_t peakKwCharge[3] = { -2.81, -2.87, -2.25 };
 	ssc_number_t peakKwDischarge[3] = { 1.39, 1.30, 0.97 };
-	ssc_number_t peakCycles[3] = { 1, 1, 1 };
-	ssc_number_t avgCycles[3] = { 1, 0.9973, 0.4904 };
+	ssc_number_t peakCycles[3] = { 1, 2, 1 };
```

Comment:
Turns out this result was different on Mac vs Linux/Windows.

In Linux, some of the batt_power entries were `-0.000` and in Mac, this was simply `0.0000`, and this resulted in a different of which cycle state was being assigned where `-0.000` was being assigned to `cycleState=-1` and triggering an extra cycle.